backend-styleguide/README.md

```diff
@@ -313,6 +313,53 @@ enumerize :gender, in: %w[
 ], scope: true
 ```
 
+### 👉 Use comments to define tables and fields
+
+We use comments to define tables and fields. This is a good approach to help newcomers and others Geeks.
+
+Don't put sensitive information, comments doesn't have any security mechanism and they are stored globally so any user connected to any database in cluster can see all comments for shared objects.
```

Comment:
```suggestion
Don't put sensitive information, comments don't have any security mechanism and they are stored globally so any user connected to any database in the cluster can see all comments for shared objects.
```

Reply:
fixed in https://github.com/GeekHunter-Brasil/hello-world/pull/12/commits/e885b0aa55ee15347b80ec92ff29b8740c957e66